alws/routers/uploads.py

```diff
@@ -19,10 +24,17 @@ async def upload_repometada(
     modules: typing.Optional[UploadFile] = None,
     comps: typing.Optional[UploadFile] = None,
     repository: str = Form(...),
+    product_name: str = Form(...),
     session: AsyncSession = Depends(get_db),
+    user: User = Depends(get_current_user),
 ):
     msg = ""
     uploader = MetadataUploader(session, repository)
+    product = get_products(session, product_name=product_name)
```

Comment:
Do we really need to add an additional field `product_name` to the endpoint? Can't we use the `product_repositories` table to find the product?

Reply:
The repository to update can also be a build repository, for example, and in this case the way to find the product will be different. Maybe just asking to provide the mane of the product is more logical

Reply:
Well, my point here is that I'd prefer to guess the product through the repo name provided by the user. The reason?
* I do not have permissions on the `AlmaLinux` product, but I own a personal product called `MyDistro`
* I can call the endpoint indicating an AlmaLinux repo (let's say `baseos_x86_64`) and indicating `MyDistro` product

The current code will allow me to upload metadata to the AlmaLinux repo even though I don't have permissions on that product.